activerecord/test/cases/schema_dumper_test.rb

```diff
@@ -19,24 +19,6 @@ def perform_schema_dump
     dump_all_table_schema []
   end
 
-  def test_dump_schema_information_with_empty_versions
-    ActiveRecord::SchemaMigration.delete_all
-    schema_info = ActiveRecord::Base.connection.dump_schema_information
-    assert_no_match(/INSERT INTO/, schema_info)
```

Comment:
We need to make sure the versions are not being written to the dump file so I'd keep those tests but inverting the assertion to make sure we have no insert statements.